XMonad/Actions/CopyWindow.hs

```diff
@@ -76,18 +77,20 @@ import qualified XMonad.StackSet as W
 -- "XMonad.Doc.Extending#Editing_key_bindings".
 
 -- $logHook
--- To distinguish workspaces containing copies of the focused window use
--- something like:
---
--- > sampleLogHook h = do
--- >    copies <- wsContainingCopies
--- >    let check ws | ws `elem` copies = pad . xmobarColor "red" "black" $ ws
--- >                 | otherwise = pad ws
--- >    dynamicLogWithPP myPP {ppHidden = check, ppOutput = hPutStrLn h}
--- >
--- > main = do
--- >    h <- spawnPipe "xmobar"
--- >    xmonad def { logHook = sampleLogHook h }
+-- To distinguish workspaces containing copies of the focused window, use the functions below.
+-- First use a check copy function that takes a pretty printer and makes it aware of copies.
+
+copiesPP :: (WorkspaceId -> String) -> PP -> X PP
+copiesPP wtoS pp = do
+    copies <- wsContainingCopies
+    let check ws | ws `elem` copies = wtoS ws 
+                 | otherwise = (ppHidden pp) ws
+    return pp { ppHidden = check }
+
+-- Then use @copiesPP@ with 'XMonad.Hooks.StatusBar.statusBarProp' :
+--
+-- > mySB = statusBarProp "xmobar" (copiesPP xmobarPP)
+-- > main = xmonad $ withEasySB mySB defToggleStrutsKey def
```

Comment:
This doesn't do what you think it does; these kinds of haddock headings have to be unboken comments.  In this case, the first part gets rendered under `$logHook`, then there's the function, and no user will ever see the bottom part.

Reply:
I forgot to hit "submit review". @oogeek try to generate the docs yourself and see how they look

Reply:
Same problem is here, btw: https://github.com/xmonad/xmonad-contrib/pull/501/files#diff-8b3eac5d6797ca9ec49c63a17e33213c05a92f32212bacd27f90e30b163c9b15R202

Reply:
Thanks for pointing out. Really sorry for my carelessness.